tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors_test.ts

```diff
@@ -0,0 +1,269 @@
+              pageLoadedSizes: {},
+              numExecutions: 10,
+            },
+            pageSize: 1000,
+            displayCount: 50,
+            focusIndex: 3,
+            scrollBeginIndex: 0,
+            executionDigests: {},
+            executionData: {
+              0: createTestExecutionData(),
+            },
+          },
+        })
+      );
+      expect(getFocusedExecutionData(state)).toBe(null);
+    });
+  });
+
+  describe('getFocusedExecutionStackFrames', () => {
+    it('returns correct stack frames when there is no focus', () => {
+      const state = createState(
+        createDebuggerState({
+          activeRunId: '__default_debugger_run__',
+          executions: {
+            numExecutionsLoaded: {
+              state: DataLoadState.LOADING,
+              lastLoadedTimeInMs: null,
+            },
+            executionDigestsLoaded: {
+              state: DataLoadState.NOT_LOADED,
+              lastLoadedTimeInMs: null,
+              pageLoadedSizes: {},
+              numExecutions: 0,
+            },
+            pageSize: 1000,
+            displayCount: 50,
+            focusIndex: null,
+            scrollBeginIndex: 0,
+            executionDigests: {},
+            executionData: {},
+          },
+        })
+      );
+      expect(getFocusedExecutionStackFrames(state)).toBe(null);
+    });
+
+    it('returns correct stack frames when there is no focus', () => {
+      const stackFrame1: StackFrame = ['localhost', '/tmp/main.py', 10, 'main'];
+      const stackFrame2: StackFrame = [
+        'localhost',
+        '/tmp/model.py',
+        20,
+        'initialize',
+      ];
+      const stackFrame3: StackFrame = [
+        'localhost',
+        '/tmp/model.py',
+        30,
+        'create_weight',
+      ];
+
+      const state = createState(
+        createDebuggerState({
+          activeRunId: '__default_debugger_run__',
+          executions: {
+            numExecutionsLoaded: {
+              state: DataLoadState.LOADING,
+              lastLoadedTimeInMs: null,
+            },
+            executionDigestsLoaded: {
+              state: DataLoadState.NOT_LOADED,
+              lastLoadedTimeInMs: null,
+              pageLoadedSizes: {},
+              numExecutions: 0,
+            },
+            pageSize: 1000,
+            displayCount: 50,
+            focusIndex: 1,
+            scrollBeginIndex: 0,
+            executionDigests: {},
+            executionData: {
+              1: createTestExecutionData({
+                stack_frame_ids: ['a1', 'a3'],
+              }),
+            },
+          },
+          stackFrames: {
+            a1: stackFrame1,
+            a2: stackFrame2,
+            a3: stackFrame3,
+          },
+        })
+      );
+      expect(getFocusedExecutionStackFrames(state)).toEqual([
+        stackFrame1,
+        stackFrame3,
+      ]);
+    });
+
+    it('returns correct stack frames when subset of frames is missing', () => {
```

Comment:
`returns correct stack frames when subset of frames is missing` -> `returns null when subset of frames is missing`